R/write.R

```diff
@@ -72,18 +72,29 @@ write_csv <- function(x, path, na = "NA", append = FALSE, col_names = !append) {
 
 #' @rdname write_delim
 #' @export
-write_excel_csv <- function(x, path, na = "NA", append = FALSE, col_names = !append) {
+write_excel_csv <- function(x, path, delim = ",", na = "NA", append = FALSE, col_names = !append) {
```

Comment:
Looks OK, but I have some doubts. If this change is going to be merged with the official `tidyverse/readr` repository, putting new argument `delim` on 3rd position introduces API breaking change. It would be safier to place it as last argument.

Legacy code like this: `write_excel_csv(data, "/path/x/z/y", "0", TRUE, TRUE)` can be dangerous after updating to the new API - it can silently assign `delim = "0"` instead `na = "0"`

But I understand normally `delim` arg is on the 3rd position. Maybe after discussion it will be accepted, but it would be good to discuss it internally first.

Reply:
good point :+1:

Reply:
thanks good point. I added that on the 3rd place so to be in line with `write_delim` function.